contrib/internal/httputil/trace_gen.go

```diff
@@ -125,6 +125,10 @@ func wrapResponseWriter(w http.ResponseWriter, span ddtrace.Span) http.ResponseW
 			monitoredResponseWriter
 			http.Hijacker
 		}{mw, hHijacker}
+	default:
+		w = struct {
+			monitoredResponseWriter
+		}{mw}
```

Comment:
The following is enough:
```suggestion
		w = mw
```